ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/SystemCallProcessor.java

```diff
@@ -69,7 +70,7 @@ public Bytes process(
     final Account maybeContract = worldState.get(callAddress);
     if (maybeContract == null) {
       LOG.trace("System call address not found {}", callAddress);
-      return null;
+      return Bytes.EMPTY;
```

Comment:
What does it mean to have a Request object with empty bytes? Is that valid to return to CL?

After a quick debug of the code, I think we'd send the CL a list with a "0x" string, which seems odd to me. Should we be omitting these from the list instead?

Reply:
We need to return these empty requests as a "0x" string to the CL currently. Returning these as empty bytes means I can just return the data as is and will also be hashed correctly for empty requests. This is probably going to change to be omitted in the devnet though.